modules/ROOT/pages/whats-new-in-dw.adoc

```diff
@@ -0,0 +1,72 @@
+= What's New in DataWeave
+
+The 2.4 version of DataWeave introduces the following features:
+
+* Larger than memory strings.
+* New reader and writer properties for working with data formats.
+* Many new modules, functions, types, annotations, and variables. Some are experimental and subject to change in future releases.
+* A number of other overloaded functions introduce new helper functions for
+handling `null` values.
+
+Changes to the documentation include more detailed function signatures and organization of the functions by their module names, such as `dw::Core`, `dw::core::Strings`, and dw::util::Math.
+
+DataWeave 2.4 is bundled with Mule 4.4. For information about the Mule 4.4 release, see xref:4.4@mule-runtime::whats-new-in-mule.adoc[What's New in Mule 4.4].
+
+== New Data Formats and Properties
+
+DataWeave introduces the following reader and writer properties:
+
+* DataWeave (dw) reader property: `privileges`
+* Excel (XLSX) reader property: `tableLimit`
+* YAML writer property: `writeDeclaration`
+* XML writer properties: `defaultNamespace`, `writeDeclaredNamespaces`
+
+== New DataWeave Function Modules and Features
+
+New DataWeave modules and features:
+
+* xref:dw-core.adoc[dw::Core] introduces new functions and annotations:
+** xref:dw-core-functions-indexof.adoc[indexOf]
+** xref:dw-core-functions-lastindexof.adoc[lastIndexOf]
+** xref:dw-core-functions-onnull.adoc[onNull]
+** xref:dw-core-functions-then.adoc[then]
+** xref:dw-core-annotations.adoc[Core Annotations] introduces a number of new annotations:
+* xref:dw-dates.adoc[dw::core::Dates] is a new module with functions and types.
```

Comment:
"functions and types to work with dates, such as {example}"

Reply:
the example could be `today`

Reply:
```suggestion
* xref:dw-dates.adoc[dw::core::Dates] is a new module with functions and types, for example, the function `today`.
```

Reply:
* xref:dw-dates.adoc[dw::core::Dates] is a new module with functions and types to create and manipulate Date types, for example, the function `today`.

Reply:
updated comment to suggested and committed it.